src/rubrix/server/tasks/text_classification/api/model.py

```diff
@@ -489,7 +489,12 @@ class TextClassificationQuery(BaseModel):
     status: List[TaskStatus] = Field(default_factory=list)
     predicted: Optional[PredictionStatus] = Field(default=None, nullable=True)
 
-    def as_elasticsearch(self) -> Dict[str, Any]:
+    only_uncovered: bool = Field(
```

Comment:
nitpicking, the `only_uncovered` is missing in the docstring.

Reply:
obsolete